frontend/src/customPropTypes/compliance.js

```diff
@@ -0,0 +1,34 @@
+import { PropTypes } from "prop-types";
+
+// This file is anticipated to have multiple exports
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
This shouldn't be needed in a file with multiple exports

Reply:
i'll remove